django-stubs/utils/timezone.pyi

```diff
@@ -30,10 +32,10 @@ class LocalTimezone(ReferenceLocalTimezone):
 
 utc: UTC = ...
 
-def get_fixed_timezone(offset: Union[timedelta, int]) -> tzinfo: ...
-def get_default_timezone() -> tzinfo: ...
+def get_fixed_timezone(offset: Union[timedelta, int]) -> FixedOffset: ...
+def get_default_timezone() -> BaseTzInfo: ...
```

Comment:
This LGTM. Even though it can't capture the exact differences between pytz `_UTCclass`, `_StaticTzInfo` and `_DstTzInfo`, it is an improvement over `tzinfo`.

Reply:
Decision to go this way was typeshed-driven. [Stubs for pytz](https://github.com/python/typeshed/blob/master/third_party/2and3/pytz/__init__.pyi) expose only that class.